### PR TITLE
[TGS-56] Created TileGameServer.BaseLibrary project

### PR DESCRIPTION
**Pull request by ArtiomCiobanu**

### CODE REVIEW

**Review by NickNovic — APPROVED** [review]

File: Source/TileGameServer/TileGameServer.Domain/TileGameServer.Domain.csproj [hunk 1]
```diff
@@ -0,0 +1,7 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+    <PropertyGroup>
+        <TargetFramework>net5.0</TargetFramework>
```
Comment (Cute-Monster) [inline Source/TileGameServer/TileGameServer.Domain/TileGameServer.Domain.csproj:4]: Add versionprefix and postfix

**Review by Cute-Monster — CHANGES_REQUESTED** [review]

File: Source/TileGameServer/TileGameServer.Domain/TileGameServer.Domain.csproj (hunk 1 above)
Comment (ArtiomCiobanu) [reply Source/TileGameServer/TileGameServer.Domain/TileGameServer.Domain.csproj:4]: Done

File: Source/TileGameServer.BaseLibrary/TileGameServer.BaseLibrary/TileGameServer.BaseLibrary.csproj [hunk 2]
```diff
@@ -0,0 +1,7 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+    <PropertyGroup>
+        <TargetFramework>net5.0</TargetFramework>
```
Comment (Cute-Monster) [inline Source/TileGameServer.BaseLibrary/TileGameServer.BaseLibrary/TileGameServer.BaseLibrary.csproj:4]: the same as was before

File: Source/TileGameServer.BaseLibrary/TileGameServer.BaseLibrary.Domain/TileGameServer.BaseLibrary.Domain.csproj [hunk 3]
```diff
@@ -0,0 +1,11 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+    <PropertyGroup>
+        <TargetFramework>net5.0</TargetFramework>
```
Comment (Cute-Monster) [inline Source/TileGameServer.BaseLibrary/TileGameServer.BaseLibrary.Domain/TileGameServer.BaseLibrary.Domain.csproj:4]: and here

**Review by Cute-Monster — CHANGES_REQUESTED** [review]

File: Source/TileGameServer.BaseLibrary/TileGameServer.BaseLibrary.Domain/TileGameServer.BaseLibrary.Domain.csproj (hunk 3 above)
Comment (ArtiomCiobanu) [reply Source/TileGameServer.BaseLibrary/TileGameServer.BaseLibrary.Domain/TileGameServer.BaseLibrary.Domain.csproj:4]: Done

File: Source/TileGameServer.BaseLibrary/TileGameServer.BaseLibrary/TileGameServer.BaseLibrary.csproj (hunk 2 above)
Comment (ArtiomCiobanu) [reply Source/TileGameServer.BaseLibrary/TileGameServer.BaseLibrary/TileGameServer.BaseLibrary.csproj:4]: Done

**Review by Cute-Monster — APPROVED** [review]

**Pull request merged**